Boyer-Moore/BoyerMoore.swift

```diff
@@ -53,15 +53,23 @@ extension String {
                 // There is a possible match. Do a brute-force search backwards.
                 if let k = backwards() { return k }
                 
-                // If no match, we can only safely skip one character ahead.
-                i = index(after: i)
+                if !useHorspoolImprovement {
+                    // If no match, we can only safely skip one character ahead.
+                    i = index(after: i)
+                }
+                else {
```

Comment:
Our style guidelines prefer to have the `else` statement on the same line as the closing brackets for the `if` statement.